docs/reference/analysis/tokenfilters/limit-token-count-tokenfilter.asciidoc

```diff
@@ -1,32 +1,134 @@
 [[analysis-limit-token-count-tokenfilter]]
-=== Limit Token Count Token Filter
+=== Limit token count token filter
+++++
+<titleabbrev>Limit token count</titleabbrev>
+++++
 
-Limits the number of tokens that are indexed per document and field.
+Limits the number of output tokens.
 
-[cols="<,<",options="header",]
-|=======================================================================
-|Setting |Description
-|`max_token_count` |The maximum number of tokens that should be indexed
-per document and field. The default is `1`
+By default, the `limit` filter keeps only the first token in a stream. For
+example, the filter can change the token stream `[ one, two, three ]` to 
+`[ one ]`.
 
-|`consume_all_tokens` |If set to `true` the filter exhaust the stream
-even if `max_token_count` tokens have been consumed already. The default
-is `false`.
-|=======================================================================
+This filter uses Lucene's
+https://lucene.apache.org/core/{lucene_version_path}/analyzers-common/org/apache/lucene/analysis/miscellaneous/LimitTokenCountFilter.html[LimitTokenCountFilter].
 
-Here is an example:
+[TIP]
+====
+The `limit` filter is commonly used to limit the size of document field values
+based on _token count_. If you want to limit the size of field values based on
```

Comment:
The first sentence in this tip seems like it belongs in the description as the "why" you want to use this filter.

Reply:
Thanks! Moved this with 64239c99fe9.